diagnostic.md

```diff
@@ -23,7 +30,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$('.listerItem:parent^2 > form')
```

Comment:
This is not a valid selector.  A couple of possible ways are `$('li').parent().parent().find('form')` and `$('li').parent().parent().children('form');`.